### PR TITLE
[ONNX] Add upsample trilinear to skip decomp

### PR DESCRIPTION
**Pull request by titaiwangms**
(1) Add upsample trilinear vec to skip decomposition
(2) Add tests to make sure that torch.export.export still decomposes them

### CODE REVIEW

**Comment by justinchuby** [comment]
> make sure that torch.export.export still decomposes them

Do you mean torch.export.export still _skips_ decomposition?

**Comment by titaiwangms** [comment]
> > make sure that torch.export.export still decomposes them
> 
> Do you mean torch.export.export still _skips_ decomposition?

The opposite. They always decompose the ops.

**Comment by justinchuby** [comment]
> > > make sure that torch.export.export still decomposes them
> > 
> > 
> > Do you mean torch.export.export still _skips_ decomposition?
> 
> The opposite. They always decompose the ops.

Curious why we want to guard this behavior then? Is it a desirable behavior?

**Comment by titaiwangms** [comment]
> > > > make sure that torch.export.export still decomposes them
> > > 
> > > 
> > > Do you mean torch.export.export still _skips_ decomposition?
> > 
> > 
> > The opposite. They always decompose the ops.
> 
> Curious why we want to guard this behavior then? Is it a desirable behavior?

Cause we are moving from our passes to torch.export.export, and after we done that, users might think that there is a regression if at the time, torch.export.export pre_dispatch still does not support "not decompose" these ops. Guarding this can remind us when these are not decomposed anymore.

File: .ci/docker/common/install_onnx.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ pip_install coloredlogs packaging
 pip_install onnxruntime==1.18
 pip_install onnx==1.16.0
 # pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@3e869ef8ccf19b5ebd21c10d3e9c267c9a9fa729" --no-deps
-pip_install onnxscript==0.1.0.dev20240523 --no-deps
+pip_install onnxscript==0.1.0.dev20240610 --no-deps
```
Comment (titaiwangms) [inline .ci/docker/common/install_onnx.sh:36]: ml_dtypes is needed.

**Review by justinchuby — APPROVED** [review]

File: .ci/docker/common/install_onnx.sh [hunk 2]
```diff
@@ -33,7 +33,9 @@ pip_install coloredlogs packaging
 pip_install onnxruntime==1.18
 pip_install onnx==1.16.0
 # pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@3e869ef8ccf19b5ebd21c10d3e9c267c9a9fa729" --no-deps
-pip_install onnxscript==0.1.0.dev20240610 --no-deps
+pip_install onnxscript==0.1.0.dev20240613 --no-deps
+# required by onnxscript
+pip_install ml_dtypes
```
Comment (justinchuby) [inline .ci/docker/common/install_onnx.sh:38]: Is it not part of the required packages of onnxscript?

Comment (justinchuby) [reply .ci/docker/common/install_onnx.sh:38]: Sorry I saw no-deps

**Comment by huydhn** [comment]
@pytorchbot revert -m 'Sorry for reverting your change but its ONNX job is failing in trunk https://hud.pytorch.org/pytorch/pytorch/commit/b72989a2b5ac4637612e31e325d7c8233fcbd7a1' -c nosignal

(The root cause is unclear yet to me)

**Comment by titaiwangms** [comment]
> @pytorchbot revert -m 'Sorry for reverting your change but its ONNX job is failing in trunk https://hud.pytorch.org/pytorch/pytorch/commit/b72989a2b5ac4637612e31e325d7c8233fcbd7a1' -c nosignal
> 
> (The root cause is unclear yet to me)

@huydhn Thanks!
I think it's fp16 low precision issue making it flaky. I have loosed the expected rtol/atol.

**Pull request closed without merging**